packages/filebrowser/src/listing.ts

```diff
@@ -897,6 +913,17 @@ export class DirListing extends Widget {
     this._startDrag(data.index, event.clientX, event.clientY);
   }
 
+  private emitOpenEvent(item: Contents.IModel): void {
```

Comment:
Private methods should have a leading underscore.  Also, this does more than emit a signal.  How about:

```typescript
/**
 * Open a file item and emit a signal.
 */
private _openItem(item: Contents.IModel): void {
```